test/extended/image_ecosystem/s2i_perl.go

```diff
@@ -46,7 +46,7 @@ var _ = g.Describe("[image_ecosystem][perl][Slow] hot deploy for openshift perl
 				oc.SetOutputDir(exutil.TestContext.OutputDir)
 
 				exutil.CheckOpenShiftNamespaceImageStreams(oc)
-				g.By(fmt.Sprintf("calling oc new-app -f %q", dancerTemplate))
+				g.By(fmt.Sprintf("calling oc new-app -f %q", dancerTemplate, "-e", "HTTPD_START_SERVERS=1", "-e", "HTTPD_MAX_SPARE_SERVERS=1", "-e", "HTTPD_MAX_REQUEST_WORKERS=1"))
```

Comment:
Shouldn't the environment variables be passed to the oc.Run command?  I think you are just printing them out here....